lib/src/collection/bimap.dart

```diff
@@ -148,9 +149,9 @@ class HashBiMap<K, V> implements BiMap<K, V> {
   }
 
   @override
-  void updateAll(V update(K key, V value)) {
+  void updateAll(V Function(K key, V value) update) {
     for (final key in keys) {
-      _add(key, update(key, _map[key]!), true);
+      _add(key, update(key, _map[key] as V), true);
```

Comment:
`_map` is defined as `Map<K, V>`, so `_map[key]` return type is `V?`. Why is `!` not sufficient here? `as V` is slower than a null check, so if it's possible to continue using `!`, we should try.

Reply:
fixed! – we should look at this lint!